app/dictionary.en-us.words

```diff
@@ -10996,6 +11008,7 @@ nowadays
 noway
 nowhere
 nowt
+ntfs
 nth
 ntp
```

Comment:
Optional (since this wasn't added in this PR, but hey I just noticed that ntp should be in ci_edit/app/dictionary.acronyms.words as well, if you're up for moving it).

Reply:
Added